src/cdk/testing/test-harnesses.md

```diff
@@ -385,7 +385,7 @@ authors to write easily understandable code, e.g.
 for the particular subclass.
 
 Harnesses that need to add additional options should extend the `BaseHarnessFilters` interface and
-additional optional properties as needed. `HarnessPredicate` provides several convenience methods
+additional optional properties as needed. `HarnessPredicate` provides several convenient methods
```

Comment:
"Convenience method" is actually a generally well-known concept: https://en.wiktionary.org/wiki/convenience_method

Reply:
Thanks for the review, I adressed the feedback